XMonad/Util/ClickableWorkspaces.hs

```diff
@@ -32,26 +33,33 @@ import XMonad.Hooks.DynamicLog (xmobarAction, xmobarRaw, PP(..))
 -- > logHook = clickablePP xmobarPP { ... } >>= dynamicLogWithPP
 --
 -- * Requirements:
---   * wmctrl on system (in path)
+--   * @xdotool@ on system (in path)
 --   * "XMonad.Hooks.EwmhDesktops" for wmctrl support (see Hackage docs for setup)
 --   * use of UnsafeStdinReader in xmobarrc (rather than StdinReader)
```

Comment:
Does this work with `UnsafeXMonadLog` as well?  I feel like it should; I will try to test this

Reply:
Yeah, wouldn't work for me otherwise. I'll update the comment.